e2e/test/scenarios/sharing/alert/email-alert.cy.spec.js

```diff
@@ -95,3 +130,25 @@ function openAlertForQuestion(id) {
 function toggleChannel(channel) {
   cy.findByText(channel).parent().find("input").click();
 }
+
+function clickAlertBell() {
+  cy.findByTestId("view-footer").within(() => {
+    cy.icon("bell").click();
+  });
```

Comment:
if you just have a single assertion, you don't need .within

```suggestion
  cy.findByTestId("view-footer")
  	.icon("bell")
  	.click();
```